src/test/java/com/amazon/opendistroforelasticsearch/knn/index/KNNJNITestsIT.java

```diff
@@ -31,8 +31,8 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
-public class KNNJNITests extends ESTestCase {
-    private static final Logger logger = LogManager.getLogger(KNNJNITests.class);
+public class KNNJNITestsIT extends ESTestCase {
```

Comment:
Why switch this?

Reply:
Ah. When I ran the test locally on IntelliJ, it fails to detect the integration tests therein due to lack of suffix "IT" in the class name. So I switched that for easier debugging. Will change it back but I think it might worth doublecheck if CI also neglects the tests.

Reply:
Tests as opposed to IT signals that it is a unit test. These tests run during `./gradlew build`, which is what we use for the PR testing [workflow](https://github.com/opendistro-for-elasticsearch/k-NN/blob/master/.github/workflows/test-workflow.yml#L26). You can also run them individually with this command:
```
./gradlew ':test' --tests "com.amazon.opendistroforelasticsearch.knn.index.KNNJNITests.testCreateHnswIndex"
```
We consider these unit tests because they are testing an isolated portion of code. Interesting that IntelliJ does not pick them up. Will need to look into it.

Reply:
Ah. Sorry, I misidentified it as IT.